src/System.Collections.Concurrent/src/System/Collections/Concurrent/ConcurrentBag.cs

```diff
@@ -513,7 +513,7 @@ public T[] ToArray()
         {
             // Short path if the bag is empty
             if (_headList == null)
-                return new T[0];
+				return Array.Empty<T>();
```

Comment:
I explicitly avoided changing this one in the past (and others in ToArray methods) because the docs for it are explicit about it creating a "new" array.